debian/control

```diff
@@ -11,7 +11,7 @@ Vcs-Browser: https://github.com/stackstorm/st2chatops
 Package: st2chatops
 Architecture: any
 Pre-Depends: dpkg (>= 1.16.16)
-Depends: nodejs (>= 4.0.0)
+Depends: nodejs (>= 6.0.0)
```

Comment:
Considering `nodejs` epopeya https://github.com/StackStorm/ops-infra/pull/206, please pin node versions to `6` for both deb/rpm.
So we won't install any node7 or node8 or whatever in future.